### PR TITLE
Improve button for expanding responses

### PR DESCRIPTION
**Pull request by Igor-J86**

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/4SntNFBJ](https://trello.com/c/4SntNFBJ)

**Comment by Igor-J86** [comment]
Skjermbilde 2022-03-01 kl  09 47 43

File: layouts/partials/api/schema.html [hunk 1]
```diff
@@ -66,7 +66,7 @@
                     </div>
                   {{- end -}}
                 </dt>
-                <dd>
+                <dd class="paxs plm">
```
Comment (halvorsanden) [inline layouts/partials/api/schema.html:69]: This causes the following as far as I can see:
Screenshot 2022-03-01 at 09 52 23

I have something for the spacing between dt/dd in the pipeline on the other task.

Comment (Igor-J86) [reply layouts/partials/api/schema.html:69]: It shows like this here...? 
Skjermbilde 2022-03-01 kl  09 56 18

File: css/documentation.css [hunk 2]
```diff
@@ -232,6 +232,12 @@ summary.error-code:hover {
   font-weight: 400;
 }
 
+.param-toggle-btn {
```
Comment (halvorsanden) [inline css/documentation.css:235]: I would have created separate classes for use in JS and CSS, so one can be changed, renamed or moved without the other one breaking.

File: layouts/partials/api/schema.html (hunk 1 above)
Comment (halvorsanden) [reply layouts/partials/api/schema.html:69]: Funky difference between Chrome and FF by the looks of it.

File: css/documentation.css (hunk 2 above)
Comment (Igor-J86) [reply css/documentation.css:235]: Ok, will change the class name and add it to the button as well. :)

File: layouts/partials/api/schema.html (hunk 1 above)
Comment (Igor-J86) [reply layouts/partials/api/schema.html:69]: But I will remove it and we can handle the spacings here and there in another task.

Comment (halvorsanden) [reply layouts/partials/api/schema.html:69]: The left padding on the dt elements without buttons cause the width of it to increase – not sure why it’s different in the two browsers though. I’d say keep it for now – I’m working on removing some of the padding to balance the different levels anyway.

**Review by halvorsanden — APPROVED** [review]

**Pull request merged**